tests/examples/message_thread_XFAIL_validation.ttl

```diff
@@ -7,6 +7,11 @@
 @prefix types: <https://ontology.unifiedcyberontology.org/uco/types/> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
+[]
```

Comment:
For everyone's awareness, this Git noise appears to be due to a new effect in `pyshacl` after `rdflib`'s 6.2.0 release.  It is a mild nuisance for Git review, but otherwise harmless.

Reply:
This should remain cleaned out, per [PR 456](https://github.com/ucoProject/UCO/pull/456).